src/screens/conclusion/index.js

```diff
@@ -2,11 +2,13 @@ import "./conclusion.css";
 
 export function Conclusion() {
   return (
-    <section id="conclusion" className="together-we-can screen-content">
+    <section id="conclusion" className="together-we-can">
       <div className="full-screen bears h-full w-full flex-col justify-between">
-        <div className="h-screen-50vh flex w-[100%] flex-col self-start py-16 pr-10 lg:w-[75%] 2xl:w-[50%]">
-          <h2>Together, we can make life better for wild animals.</h2>
-          <p>
+        <div className="h-screen-50vh py-30 flex w-[100%] max-w-3xl flex-col pt-36 pl-28 text-white lg:w-[75%] 2xl:w-[50%]">
```

Comment:
```suggestion
        <div className="h-screen-50vh py-30 flex w-full max-w-3xl flex-col pt-36 pl-28 text-white lg:w-3/4 2xl:w-1/2">
```

Not sure about the usage of `h-screen-50-vh` and similar things like percentages. Will this look okay when resizing the screen, both vertically and horizontally?